tcms/signals.py

```diff
@@ -209,19 +209,33 @@ def handle_comments_pre_delete(sender, **kwargs):
 def handle_emails_post_bug_save(sender, instance, created=False, **kwargs):
     """
     Send email updates to assignee after they've been
-    assigned a bug on bug creation.
+    assigned a bug on bug creation or there is a new comment to the bug.
     """
-    if not created or instance.assignee is None:
+    from tcms.core.helpers.comments import get_comments
+
+    comments = get_comments(instance)
+
+    if instance.assignee is None or comments.count() < 1:
+        return
+
+    if (not instance.status and comments.last().comment == _("*bug closed*")) or (
+        instance.status and comments.last().comment == _("*bug reopened*")
+    ):
         return
 
+    recipients = [instance.assignee.email, instance.reporter.email]
```

Comment:
Potentially recipients should be anyone who has commented on the bug, reported it or is assigned to it b/c they may be interested to know about the new updates. 

Since you are modifying existing behavior and making it more broad it makes sense to update the list of recipients as well.